kotlinx.fuzz.api/build.gradle.kts

```diff
@@ -1,12 +1,15 @@
 plugins {
     id("kotlinx.fuzz.src-module")
+    `kotlin-dsl`
```

Comment:
You are not using `kotlin-dsl`. So, please, remove it.
Insted add `implementation(gradleApi())` to dependencies